src/main/scad/laszlo-molnar/l-i-vator-i/molnar.l-i-vator-i.scad

```diff
@@ -0,0 +1,86 @@
+include <puzzlecad.scad>
+
+box_puzzle_border = 6;
+
+$burr_inset = 0.125;
+$burr_bevel = 1;
+$unit_beveled = true;
+$burr_scale = 17;
+$joint_inset = 0.06;
```

Comment:
Starting with 2.0 puzzlecad will have a default $joint_inset of 0.015, which I've found strikes a good balance. 0.06 will make them super loose on many printers (on my printer, I'd have to use glue for almost all joints).

I'd suggest we not declare $joint_inset explicitly in the .scad files, and just go with the defaults (anyone can still apply a custom $joint_inset for their own prints, of course).

Reply:
Ok, got it